File: Java/205. Isomorphic Strings.java
class Solution {
    public boolean isIsomorphic(String s, String t) {
        int ms[]=new int[256];
        int mt[]=new int[256];
        for (int i = 0; i < s.length(); i++) {
            if (ms[s.charAt(i)] != mt[t.charAt(i)]){
                return false;
            }
            ms[s.charAt(i)] = i + 1;
            mt[t.charAt(i)] = i + 1;
        }
        return true;
    }
}

File: JavaScript/205. Isomorphic Strings.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isIsomorphic = function(s, t) {
    var ms=new Array();
    var mt=new Array();
    for (var i = 0; i < s.length; i++) {
        if (ms[s[i]] != mt[t[i]]){
            return false;
        }
        ms[s[i]] = i + 1;
        mt[t[i]] = i + 1;
    }
    return true;
};
